doc/release/1.12.0-notes.rst

```diff
@@ -148,5 +148,8 @@ Deprecations
 
 Assignment of ndarray object's ``data`` attribute
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Assigning the 'data' attribute is an inherently unsafe operation as pointed out
-in gh-7083. Such a capability will be removed in the future.
+Assigning the 'data' attribute is an inherently unsafe operation as pointed out in gh-7083. Such a capability will be removed in the future.
```

Comment:
Keep lines < 80 characters long, this one is 142 ;) I'll fix it up later, but keep that in mind.